flight/Modules/Autotune/autotune.c

```diff
@@ -61,6 +61,7 @@ enum AUTOTUNE_STATE { AT_INIT, AT_START, AT_RUN, AT_FINISHED, AT_WAITING };
 struct at_queued_data {
 	float y[3];	/* Gyro measurements */
 	float u[3];	/* Actuator desired */
+	float t;	/* Throttle desired */
```

Comment:
It wouldn't be a killer to give this another name? t is usually reserved for time and this will be confusing when reading code below.